ddtrace/internal/datadog/profiling/dd_wrapper/src/interface.cpp

```diff
@@ -99,12 +99,19 @@ ddup_config_sample_type(unsigned int _type) // cppcheck-suppress unusedFunction
 {
     Datadog::SampleManager::add_type(_type);
 }
+
 void
 ddup_config_max_nframes(int max_nframes) // cppcheck-suppress unusedFunction
 {
     Datadog::SampleManager::set_max_nframes(max_nframes);
 }
 
+void
+ddup_config_timeline_enabled(bool enabled) // cppcheck-suppress unusedFunction
```

Comment:
Grammatically, I'd call this `enable_timeline` rather than the current, which to me implies a predicate?